src/components/Auth/components/Login/Login.tsx

```diff
@@ -0,0 +1,79 @@
+import { useFormik } from "formik";
+import * as React from "react";
+import * as Yup from "yup";
+import { Button, Typography } from "@mui/material";
+import * as Styles from "./styles";
+
+const ValidateFormSchema = Yup.object().shape({
+  username: Yup.string()
+    .min(6, "Pocos caracteres")
+    .max(10, "Muchos caracteres")
+    .required("Username is required"),
+  password: Yup.string()
+    .min(6, "Pocos caracteres")
+    .max(10, "Muchos caracteres")
+    .required("Password is required")
+});
+
+export default function Login() {
+  const formik = useFormik({
+    initialValues: {
+      username: "",
+      password: ""
+    },
+    validationSchema: ValidateFormSchema,
+    onSubmit: (value) => {
+      console.log({ value });
+    }
+  });
+  console.log(formik);
+  return (
+    <Styles.Element>
+      <Typography
+        component="h2"
+        variant="h3"
+        align="center"
+        fontWeight="bold"
+        color="thertiary"
+        marginBottom={"15px"}
+      >
+        Login
+      </Typography>
+      <form onSubmit={formik.handleSubmit}>
+        <Styles.CustomTextField
+          variant="standard"
+          required
+          fullWidth
+          name="username"
+          label="Username"
+          onChange={formik.handleChange}
+          /* helperText={formik.errors.user}
```

Comment:
```suggestion
          helperText={formik.errors && formik.errors.username }
```
Puedes aprovechar mas lo que trae por defecto MUI para mostrar el feedback al usuario